guess-the-number/guess_the_number/guess.py

```diff
@@ -146,6 +146,20 @@ async def make_base_ui(q):
         icon='ChatBot',
         icon_color=WaveColors.cyan,
     )
+    q.page['user'] = ui.header_card(
```

Comment:
Let's reuse the existing header and make something like:

<img width="1660" alt="image" src="https://user-images.githubusercontent.com/64769322/194805027-c637283d-2796-4626-9d77-b901deb66104.png">